src/antsibull_build/pypi.py

```diff
@@ -86,7 +86,6 @@ class ReleaseInfo(p.BaseModel):
 
 
 class UrlInfo(p.BaseModel):
-    comment_text: str
     digests: dict[str, str]
     filename: str
     md5_digest: str
```

Comment:
@gotmax23 maybe we should remove all other fields we're not interested in? This isn't the first time that some unrelated field changed / got removed and creating the announcements failed because of it.